portal-impl/src/com/liferay/portal/service/impl/RoleLocalServiceImpl.java

```diff
@@ -2118,6 +2109,24 @@ protected void validate(
 		}
 	}
 
+	private int _getAssigneesTotalWithActiveUser(
+			List<UserGroupRole> userGroupRoles)
+		throws NoSuchUserException {
+
+		int assigneesTotal = 0;
+
+		for (UserGroupRole userGroupRole : userGroupRoles) {
```

Comment:
this solution has performance issue by getting all user one by one.

Reply:
It just gets users with same roleId